games-fps/xonotic/xonotic-0.8.6-r1.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop check-reqs toolchain-funcs xdg
+
+DESCRIPTION="Fork of Nexuiz, Deathmatch FPS based on DarkPlaces, an advanced Quake 1 engine"
+HOMEPAGE="https://xonotic.org/"
+SRC_URI="https://dl.xonotic.org/${P}.zip"
+S="${WORKDIR}/${PN^}"
+
+LICENSE="GPL-3+ GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="X +alsa ode +sdl"
+
+# note: many dependencies are used through dlopen()
+COMMON_UIDEPEND="
+	media-libs/libogg
+	media-libs/libtheora
+	media-libs/libvorbis"
+RDEPEND="
+	dev-libs/d0_blind_id
+	media-libs/libjpeg-turbo:=
+	media-libs/libpng
+	media-libs/freetype:2
+	<net-misc/curl-8.10.0
```

Comment:
Previous comment aside, please git mv, there's no reason to keep -r0 for a dependency change. Furthermore portage could "potentially" choose to resolve the conflict by not updating xonotic rather than downgrading curl.

I'd also add a comment above RDEPEND pointing to the downstream bug to know if it can be removed.